cmake/projects/SocketIOClient/hunter.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2013-2017, Ruslan Baratov, Richard Hodges
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+        PACKAGE_NAME
+        SocketIOClient
+        VERSION
+        "1.6.1"
+        URL
+        "https://github.com/madmongo1/socket.io-client-cpp/archive/v1.6.1-hunter.tar.gz"
```

Comment:
> Should I re-apply my changes to this fork?

Yes, just push your branch to this fork and make same release. Also see docs:
* https://docs.hunter.sh/en/latest/creating-new/cmake-dependencies.html#maintenance